lib/llm/provider.rb

```diff
@@ -21,6 +21,14 @@ def initialize(secret, host, port = 443)
       end
     end
 
+    ##
+    # Returns an inspection of the provider object
+    # @return [String]
+    # @note The secret key is redacted in inspect for security reasons
+    def inspect
+      "#<#{self.class.name} http: #{@http.inspect} secret=[REDACTED]>"
```

Comment:
We should prefix instance variables with `@` and use `=` to separate key and value:
```ruby
@secret=[REDACTED] @http={@http.inspect}
```
What do you think about we explore `PP` as a means to generate the `inspect` output ?

Reply:
oups! i missed that.
i'm down for PP, just not sure what's a good practice for using it.

Reply:
It can help you build the inspect string, it can be called directly from the `inspect` method.